sdk/core/Azure.Core/src/IJsonSerializable.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+
+namespace Azure
+{
+    /// <summary>
+    /// TODO
+    /// </summary>
+    public interface IJsonSerializable
+    {
+        /// <summary>
+        /// TODO
```

Comment:
Please create tracking issues for these TODOs if not already present.

Reply:
Ahh I just saw it is in your private branch. It may be still helpful for you to track when you create a public PR.